tests/test_caiman_segmentation_extractor.py

```diff
@@ -86,6 +87,357 @@ def test_caiman_segmentation_extractor():
     for image in available_images.values():
         assert image.shape == frame_shape
 
+    # Test quality metrics (backward compatibility)
+    snr_values = extractor._get_snr_values()
+    expected_snr = np.array(
+        [
+            2.91673987,
+            1.05541609,
+            1.59315755,
+            4.66348506,
+            2.97824475,
+            2.98275578,
+            2.9776668,
+            2.94389109,
+            1.19369399,
+            1.76922279,
+        ]
+    )
+    assert_array_almost_equal(snr_values, expected_snr, decimal=6)
+    r_values = extractor._get_spatial_correlation_values()
```

Comment:
Now that we have properties we don't need the test the private methods. The tests that you have below that test that the quality metrics can be extracted as properties and comparing them to these expected values are enough I think.